mw-webapp/vite.config.ts

```diff
@@ -3,6 +3,7 @@ import {defineConfig, loadEnv} from "vite";
 import eslint from "vite-plugin-eslint";
 import viteTsconfigPaths from "vite-tsconfig-paths";
 
+// eslint-disable-next-line no-restricted-exports
 export default defineConfig(() => {
```

Comment:
could you exalude this file from linting and remove comment please?

Reply:
done